src/components/CallToAction.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { css } from 'emotion';
+import { textSans } from '@guardian/src-foundations/typography';
+import { palette } from '@guardian/src-foundations';
+
+const link = css`
+    background: ${palette.brandYellow.main};
+    border-radius: 21px;
+    box-sizing: border-box;
+    color: ${palette.neutral[7]};
+    ${textSans.medium({ fontWeight: 'bold' })};
+    text-decoration: none;
+    padding: 12px 46px 12px 21px;
+    line-height: 18px;
+    position: relative;
+    margin: 4px 10px 4px 0;
```

Comment:
We might be able to use the `space.ts` stuff from src-foundations here (and for the padding too/generally in the PR). It doesn't look like there are exact equivalents for every case, but probably close enough that people won't notice.